src/Sulu/Bundle/MediaBundle/Media/Storage/StorageInterface.php

```diff
@@ -19,45 +19,23 @@
 {
     /**
      * Save the document in a storage and give back the path to the document.
-     *
-     * @param $tempPath
-     * @param $fileName
-     * @param $version
-     *
-     * @return mixed
      */
-    public function save($tempPath, $fileName, $version);
+    public function save(string $tempPath, string $fileName, int $version): string;
```

Comment:
storage should not know about versions thats why we removed it in #4024 https://github.com/sulu/sulu/pull/4024/files#diff-d1603c50279a10b85948f4e515ee33c4